resources/schema.json

```diff
@@ -111,22 +111,24 @@
             }
           },
           "image": {
-            "title": "Bild",
             "type": "object",
+            "Q:type": "files",
             "Q:options": {
               "compact": true,
               "availabilityChecks": [
                 {
                   "type": "UserHasRole",
                   "role": "expert-election-executive"
                 }
-              ]
-            },
-            "properties": {
-              "base64": {
```

Comment:
you can use
```
"Q:options": {
  "hideInEditor": true
}
```
to hide this in editor but have it still in the schema. otherwise all the existing ones will fail because they do not validate against the schema anymore.
That way you also do not have to do a major release as it's not breaking.